Remotion/Data/DomainObjects/DomainImplementation/Transport/XmlTransportItem.cs

```diff
@@ -146,6 +146,11 @@ private void SerializePropertyValue (XmlWriter writer, PropertyDefinition? prope
         {
           writer.WriteString(((IExtensibleEnum)value).ID);
         }
+        else if (IsDateOnlyType(Assertion.IsNotNull(valueType)))
+        {
+          var dateOnly = value as DateOnly?;
+          writer.WriteString(dateOnly.ToString());
```

Comment:
please add format-identifier here and when parsing (if DateOnly.Parse also has an overload). We should check which format-specifier is the standard date format `yyyy-mm-dd`. Right now, we would format using the thread culture and might cause all sorts of mischief.

Reply:
I think we want `yyyy-dd-mm`, this is what the DB receives from DateTime queries.

Reply:
Ah nevermind, was in the wrong table still, which had a value with month and day exactly switched.